Categories/Category/Instance/Groupoids.agda

```diff
@@ -0,0 +1,72 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Category.Instance.Groupoids where
```

Comment:
This part should definitely be pulled out of this PR - I know you've already done so, but it also means I can't do much with this PR as is.